Scribe-i18n/Scripts/convert_jsons_to_xcstrings.py

```diff
@@ -25,38 +25,29 @@
     )
 
     for lang in languages:
-        if lang != "en-US":
-            lang_json = json.loads(
-                open(os.path.join(directory, f"{lang}.json"), "r").read()
+        lang_json = json.loads(
+            open(os.path.join(directory, f"{lang}.json"), "r").read()
+        )
+
+        if key in lang_json:
+            translation = lang_json[key].replace('"', '\\"').replace("\n", "\\n")
+        else:
+            translation = ""
+
+        if lang == "en-US":
+            lang = "en"
+        if translation != "":
+            data += (
+                f'        "{lang}" : {{\n'
+                f'          "stringUnit" : {{\n'
+                f'            "state" : "",\n'
+                f'            "value" : "{translation}"\n'
+                f"          }}\n"
+                f"        }},\n"
```

Comment:
Just wondering something about a potential future evolution for the script..

If I've understood, `Localizable.xcstrings` is essentially structured in JSON-like format as well? (correct me if I'm mistaken though). If so, would we be able to more heavily lean into using the `json` Python module to help with its creation? I'm mostly thinking like as in using Python objects/dicts to help build the full JSON for `Localizable.xcstrings` and then using `json.dumps()` to get the content to populate the file at the end (as in [ref](https://realpython.com/python-json/#writing-json-with-python)). Just thinking it could be a way to use some clearer, simpler Python code as opposed to having to format the file by manually adjusting for spaces and newlines, escaping characters, etc. 

Would this make sense?
If so, can absolutely be something for later - as the current version appears to be largely working. Just something that came to mind here.
CC @andrewtavis @Jag-Marcel